app/pandas.ipynb

```diff
@@ -208,18 +276,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
-   "source": [
-    "# Write your code here"
-   ],
+   "source": "df[\"Average age\"] = df[\"Age\"].mean() ",
```

Comment:
In Task 4.2, adding 'Average age' as a column with the same value for all rows may not be meaningful. Consider whether this is the intended use or if it should be calculated differently.